client/core/core_test.go

```diff
@@ -5240,10 +5241,12 @@ func TestResolveActiveTrades(t *testing.T) {
 			t.Fatalf("%s: expected %d coin loaded, got %d", description, expCoinsLoaded, len(trade.coins))
 		}
 
+		isActive := dbOrder.MetaData.Status == order.OrderStatusBooked || dbOrder.MetaData.Status == order.OrderStatusEpoch
+
 		if lo.T.Sell && ((match.Side == order.Taker && match.Status < order.MatchComplete) ||
 			(match.Side == order.Taker && match.Status < order.MakerRedeemed)) {
-			var reReserveQty uint64 = redemptionReserves
-			if dbOrder.MetaData.Status > order.OrderStatusBooked {
+			var reReserveQty uint64
+			if isActive {
```

Comment:
Not sure what was going on with this test. We only want locked if the matches are active correct?

Reply:
Ok, I got it now. We want them all locked unless the match is no longer active then just the matched. Took a while to seep in.